modules/websocket/wsl_client.h

```diff
@@ -63,10 +71,15 @@ class WSLClient : public WebSocketClient {
 
 	String _key;
 	String _host;
-	int _port;
-	Array ip_candidates;
+	uint16_t _port;
+	String _path;
+	Array _ip_candidates;
 	Vector<String> _protocols;
+	Vector<String> _original_protocols;
```

Comment:
I don't understand why we need the `_original_protocols`.
It seems there is already the `_protocols` variable that could be used.